examples/k8s-patch-basic.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: k8s-patch-basic-
+  labels:
+    workflows.argoproj.io/test: "true"
+  annotations:
+    workflows.argoproj.io/description: |
+      This example patches
```

Comment:
more detail?

Reply:
I just copied the existing patch verbatim.  I'll add a bit more.

Reply:
added